backend/apps/core/management/commands/restore_db.py

```
@@ -0,0 +1,51 @@
+            '-p', str(db_port),
+            '-U', str(db_user),
+            '-d', str(db_name),
+            '--clean',  # Drop objects before creating
+            '--if-exists',
+            '-1', # Single transaction
+            str(backup_file)
+        ]
+        
+        self.stdout.write(self.style.NOTICE(f'Starting restore of database {db_name} from {backup_file}...'))
+        try:
+            subprocess.run(command, env=env, check=True)
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)